<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdd</class>
 <widget class="QDialog" name="DialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operator editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mcc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
alternate-background-color: rgb(120, 120, 120);</string>
   </property>
   <property name="text">
    <string>MCC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mnc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>MNC</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_mnc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
alternate-background-color: rgb(120, 120, 120);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_mcc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon_operator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/icons/ques.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="icon_country">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/icons/ques.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
